x-pack/plugins/security_solution/server/fleet_integration/handlers/install_prepackaged_rules.ts

```diff
@@ -37,6 +39,8 @@ export const installPrepackagedRules = async ({
   securityStart,
   alerts,
   maxTimelineImportExportSize,
+  prebuiltRulesFromFileSystem,
```

Comment:
Just for my own understanding - Is this why we needed to "thread" these two new arguments all the way through to the endpoint context + Fleet integration callback? to feed it into `alerts.getAlertsClientWithRequest()`?

Reply:
yeah exactly